modules/swagger-codegen/src/main/resources/typescript-fetch/api.mustache

```diff
@@ -0,0 +1,309 @@
+// tslint:disable
+{{>licenseInfo}}
+
+import * as url                                 	    from "url";
+import * as isomorphicFetch                     	    from "isomorphic-fetch";
```

Comment:
Do we need to introduce `isomorphic-fetch` as a dependency? Why not inject the fetch implementation from the outside instead?